blocks/library/button/style.scss

```diff
@@ -37,3 +37,9 @@ $blocks-button__line-height: $big-font-size + 6px;
 		text-align: right;
 	}
 }
+
+.wp-block-button .wp-block-button__link {
```

Comment:
There already is an appropriate selector within this file, the `white-space` and `word-break` rules should be updated there directly.  As for their merit, I'll loop in @jasmussen.

Reply:
Thanks @mcsf for suggesting more specific way to add css. I have added my css by your suggested way.